### DISCUSSION TITLE
Could I suggest the website quasar.dev use a light background instead?

### DISCUSSION THREAD

**Discussion by sgon00** [quasarframework/quasar#17054] [Ideas / Proposals]
Hi, I am new to quasar and really like it. Could I suggest the website quasar.dev use a light background instead? The reason why I am saying this is because I want quasar to be better and get known by more people. The current black dark background makes the website feel like a kid toy project.

**Comment by ivanjaros** [comment]
you can toggle the day/night mode on top right corner. i too hate the dark UIs.

**Reply by sgon00** [reply]
Thanks a lot for the reply, but I can't find the toggle button on the top right corner (running chrome on linux). 
And even if there is a toggle button, I think the light UI should be the default, otherwise the website feels like a kid toy project to someone seeing it for the first time.

**Reply by sgon00** [reply]
Btw, I found the toggle button in other non-index pages, but not on the index page (https://quasar.dev/)

**Reply by ivanjaros** [reply]
i see they changed the website again. it is still in the the upper right corner but now it is just an icon with no tooltip or anything.
